### PR TITLE
Release go sdk

### PR DESCRIPTION
**Pull request by bill-mahoney**
Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by lenny-intel — APPROVED** [review]

File: release/app-functions-sdk-go.yaml [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-commitId: '521ab93fa6410c1be0cb80b1cb635940af6f602b'
+commitId: 'af808cf386d9f357a83198a2b4bd92a72b60b8ea'
 gitTag: true
 dockerImages: false
-gitHubRelease: false
\ No newline at end of file
+gitHubRelease: false
+apiInfo: [
+   nextReleaseVersion: '2.4.0',
+   reviewers: edgexfoundry/edgex-committers
```
Comment (ernestojeda) [inline release/app-functions-sdk-go.yaml:13]: The `edgex-committers` team seems a bit out of date. It contains a few members who are no longer active in the project. I suggest: `edgexfoundry/app-functions-sdk-go-committers`

File: release/device-sdk-go.yaml [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
-commitId: '4f64ab83298cb604fe88218673904389f4010661'
+commitId: 'b76771b07adf0b8c2e38dd8a749adeac8b55b549'
 gitTag: true
 dockerImages: false
-gitHubRelease: false
\ No newline at end of file
+gitHubRelease: false
+apiInfo: [
+   nextReleaseVersion: '2.4.0',
+   reviewers: edgexfoundry/edgex-committers
```
Comment (ernestojeda) [inline release/device-sdk-go.yaml:13]: The edgex-committers team seems a bit out of date. It contains a few members who are no longer active in the project. I suggest: `edgexfoundry/device-sdk-go-committers`

**Review by ernestojeda — CHANGES_REQUESTED** [review]
See inline comments

**Review by lenny-intel — APPROVED** [review]
LGTM

File: release/device-sdk-go.yaml (hunk 2 above)
Comment (lenny-intel) [reply release/device-sdk-go.yaml:13]: All our committer lists need cleaning up. :-(

**Review by lenny-intel — APPROVED** [review]

File: release/app-functions-sdk-go.yaml [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+commitId: 'af808cf386d9f357a83198a2b4bd92a72b60b8ea'
 gitTag: true
 dockerImages: false
-gitHubRelease: false
\ No newline at end of file
+gitHubRelease: false
+apiInfo: [
+   nextReleaseVersion: '2.4.0',
+   reviewers: edgexfoundry/app-functions-sdk-go-committers
+]
```
Comment (ernestojeda) [inline release/app-functions-sdk-go.yaml:14]:
```suggestion
apiInfo:
   nextReleaseVersion: '2.4.0',
   reviewers: edgexfoundry/app-functions-sdk-go-committers
```

File: release/device-sdk-go.yaml [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+commitId: 'b76771b07adf0b8c2e38dd8a749adeac8b55b549'
 gitTag: true
 dockerImages: false
-gitHubRelease: false
\ No newline at end of file
+gitHubRelease: false
+apiInfo: [
+   nextReleaseVersion: '2.4.0',
+   reviewers: edgexfoundry/device-sdk-go-committers
+]
```
Comment (ernestojeda) [inline release/device-sdk-go.yaml:14]:
```suggestion
gitHubRelease: false
apiInfo:
   nextReleaseVersion: '2.4.0',
   reviewers: edgexfoundry/device-sdk-go-committers
```

**Review by lenny-intel — APPROVED** [review]

**Pull request merged**